src/peergos/shared/user/TodoBoard.java

```diff
@@ -0,0 +1,84 @@
+package peergos.shared.user;
+
+import jsinterop.annotations.JsType;
+import peergos.shared.cbor.CborObject;
+import peergos.shared.cbor.Cborable;
+
+import java.time.LocalDateTime;
+import java.time.ZoneOffset;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@JsType
+public class TodoBoard implements Cborable {
+
+    private static final String VERSION_1 = "1";
+    private final List<TodoList> todoLists;
+    private final String name;
+    private final LocalDateTime timestamp;
```

Comment:
timestamp of what?

Reply:
The TodoBoard. This is for change detection. If A shared a todo board with write access to B. A or B can smash the changes made by the other.  This highlighted a bug in overwriteFile() whereby the file properties were not been updated correctly

Reply:
We should prevent overwriting changes the same way we do with the filewriter, using the CAS on file version.